index.js

```diff
@@ -11,13 +11,15 @@ const plugin = {
     rules: {
         property: require("./lib/rules/property"),
         method: require("./lib/rules/method"),
+        parsing_method: require("./lib/rules/parsing_method"),
     },
     configs: {},
 };
 
 const rules = {
     "no-unsanitized/property": "error",
     "no-unsanitized/method": "error",
+    "no-unsanitized/parsing_method": "warn",
```

Comment:
@mozfreddyb I'm glad that I did actually give it an explicit try locally, this was meant to be "warn" instead of "warning". It wasn't covered by any test and so easy to miss, I thought to take a look into adding at least a very basic integration test (one that would have caught the issue in the PR and before releasing a new version).

We can also decide to move that proposed addition to the tests and CI job in a separate PR if we prefer.

Reply:
I think it would be good to have tests in the same moment that we add the functionality. :)

Reply:
100%, I couldn't agree more